applications/nrf_desktop/doc/dev_descr.rst

```diff
@@ -0,0 +1,28 @@
+.. _dev_descr:
+
+Device Description
+##################
+
+The ``dev_descr`` module defines custom GATT Service, which contains information if the Peripheral supports the LLPM.
+Only the nrfxlib Link Layer (:option:`CONFIG_BT_LL_NRFXLIB`) supports the LLPM extension.
+The Service is mandatory for all the nRF Desktop Peripherals that should connect to the nRF Desktop Centrals.
```

Comment:
```suggestion
The Service is mandatory for all nRF Desktop peripherals that connect to the nRF Desktop centrals.
```
I understand that you must use it with the peripherals that require the centrals, but correct me if I'm wrong.
Also, let's keep centrals decapitalized.

Reply:
To clarify:
The nRF Destkop peripherals can connect with nRF Desktop centrals or with other centrals (like Android phone, Windows 10 laptop etc.). Only nRF Desktop centrals are reading the description provided by this module. nRF Desktop centrals require mentioned description to check if they can switch to LLPM connection parameters, other centrals do not support the LLPM.